web/controllers/courseController/utils.ts

```diff
@@ -1,7 +1,6 @@
-import { ApiCourse, ApiSubcourse } from "./format";
+import {ApiCourse, ApiSubcourse} from "./format";
```

Comment:
I understand your point. Yes, it's a configuration of IntelliJ to autoformat the code. I will reconfigure my IDE to prevent the mass on code changes by formatting to make the review easier. In the future I will have a look not to push formatting code changes.